pymks/mks_structure_analysis.py

```diff
@@ -1,6 +1,6 @@
 from pymks.stats import correlate
 from sklearn.base import BaseEstimator
-from sklearn.decomposition import RandomizedPCA
+from sklearn.decomposition import PCA
```

Comment:
Did you mean to change this to `PCA`?

Reply:
Yes, looks like I missed this at several places